packages/runner/unified-runner.tsx

```diff
@@ -5,7 +5,6 @@ import { Reporter } from '@packages/reporter/src/main'
 import shortcuts from '@packages/reporter/src/lib/shortcuts'
```

Comment:
Nuked above spec file - there's a ton of stubbing and whatnot, I'm not a huge fan of unit testing stores anyway... we will def need coverage, but now we've got Cypress in Cypress that will help a lot with Studio tests. I will also look to add some for the more complex logic once we've got the whole UI implemented.